libc/src/__support/FPUtil/FPBits.h

```diff
@@ -435,6 +445,13 @@ struct FPRepSem : public FPStorage<fp_type> {
   LIBC_INLINE constexpr bool is_normal() const {
     return is_finite() && !is_subnormal();
   }
+
+  // Modifiers
+  LIBC_INLINE constexpr void next_toward_inf() {
```

Comment:
How is this going to be used ? It would maybe make sense to return a new value rather than modify ?

Reply:
We will construct an `FPRep` and depending on `condition` we will return it as-is or the `next_toward_inf` value.
We _could_ return a new value instead of modifying it.

Reply:
In that case it probably makes more sense to return by value ?

IIUC, this would look like:
```
const FPRep input = ...;
return cond ? input : input.next_toward_inf()
```

instead of 

```
FPRep input = ...;
if (cond) {
   input.next_toward_inf();
}
return input;
```

Reply:
Done